### ISSUE TITLE
Add, Remove, or Move fields on Registration Page

### ISSUE THREAD

**Issue by kathypham-gov** [department-of-veterans-affairs/SCIInterim#353]
Will make a separate issue for the cosmetic changes of changing the label name. These changes below involve moving around fields, or updating the data model. All these are from the updated spreadsheet: 
https://github.com/department-of-veterans-affairs/SCIInterim/blob/master/doc/SCIDO%20data%20fields-edited%2009-02-2015.xlsx
# Add or Remove
- (N5) Computed Age 
- (N15) Email address in Patient Contact 
- (N23) Date of Expired in VA Status, next to "Status" 
- (N36) Email address for NON-VA Primary Care Physican 
- Peter removed "Date of Arrival" but I don't remember if MIOs liked this or not
# Update:

Our sections differ from the current sections in the database. Current sections (see spreadsheet):
(N1) Personal Information
(N8) Contact Information
(N20) Travel Eligibility
(N22) VA Status
(N27) Primary Care
(N39) SCI/D Information
(N43) ISNCSCI
(N56) Psychsocial
(N59) Residency and Caregiver

(N7) SCIDO ID - make this a not-editable field. Also, it is called CDW Patient ID. 

(N18) Move **Theatre of Service** and **Currently on Active Duty** to "Contact Information" - Right now it is in the SCI/D Information section. 

**Travel Eligibility**: 
- (N20) Only have "Travel Status" and "Benefits Exemption Date" in this section. 
- (N20-26) Move everything to a "VA Status" Section and include all the "SCI Hub and Physician Information".  Change name of "SCI Hub and Physician Information" to  "VA Status" 
- (N20) Travel Eligibiity status fields are a little different. The new spreadsheet combined "30% and "VA pension": (Not eligible),(>=30% service connected or VA pension), (beneficiary travel waiver)
  It will be easier to look at N20-N26 (See column F) in the spreadsheet:
  https://github.com/department-of-veterans-affairs/SCIInterim/blob/master/doc/SCIDO%20data%20fields-edited%2009-02-2015.xlsx

**VA Status** : 
(N22-26) Right now, we have a section called "SCI Hub and Physician." Change to "VA Status" and add the following: 
- (N22) Status 
- (N23) Date of Expired (New Field)
- (N24) What is the Veteran's preferred SCI Center? ( 24 centers list) This is currently "Preferred SCI Hub"
- (N25)What is the nearest VA in relation to the Veteran's residence? (VAMC list) This is currently "Assigned VAMC"
- (N26) What is the Veteran's assigned SCI Center based on the Veteran's residency? (24 centers list) This is currently "Assigned SCI Hub"

**Primary Care**
(N27) Update these answer choices: Where does the Veteran receive day-to-day primary care?:  
- At the preferred SCI Center)
- At another VA/CBOC
- At a Non-VA facility

(N28) VA Facility:
Peter added conditional here for the "What is the name of the facility" section. What happens here depends on N27. 
Can you look at N28-38?

**SCI/D Information**
(N39) SCI type renamed to "Classification of SCI Injury" with the following choices: 
- High Tetra (C1-C4, A,B,C)
- Low Tetra (C5-C8, A,B,C)
- Para (A,B,C)
- ASIA D (at Start

**Comment by awong-dev** [comment]
@kathytpham: Classification is a small change...just fixing text choices.  FYI, I'm assigning this back at you per discussion in the 9/8 weekly meeting.  Please break apart into two issues: "required for feature complete" and "nice to have".

**Comment by awong-dev** [comment]
obsoleted after reconciling from access database.
